DevUIArcJsonRPCTest.testInvocations unstable

It sometimes fail with the following error:

```
2023-07-01T02:27:55.6114078Z 2023-07-01 02:27:55,541 ERROR [io.qua.dev.run.jso.JsonRpcCodec] (vert.x-eventloop-thread-0) Error in JsonRPC Call: java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
2023-07-01T02:27:55.6115418Z 	at io.smallrye.mutiny.unchecked.UncheckedSupplier.lambda$toSupplier$0(UncheckedSupplier.java:45)
2023-07-01T02:27:55.6116700Z 	at io.smallrye.context.impl.wrappers.SlowContextualSupplier.get(SlowContextualSupplier.java:21)
2023-07-01T02:27:55.6117526Z 	at io.smallrye.mutiny.operators.uni.builders.UniCreateFromItemSupplier.subscribe(UniCreateFromItemSupplier.java:28)
2023-07-01T02:27:55.6118213Z 	at io.smallrye.mutiny.operators.AbstractUni.subscribe(AbstractUni.java:36)
2023-07-01T02:27:55.6118790Z 	at io.smallrye.mutiny.groups.UniSubscribe.withSubscriber(UniSubscribe.java:51)
2023-07-01T02:27:55.6119324Z 	at io.smallrye.mutiny.groups.UniSubscribe.with(UniSubscribe.java:110)
2023-07-01T02:27:55.6119809Z 	at io.smallrye.mutiny.groups.UniSubscribe.with(UniSubscribe.java:88)
2023-07-01T02:27:55.6120320Z 	at io.quarkus.devui.runtime.comms.JsonRpcRouter.route(JsonRpcRouter.java:199)
2023-07-01T02:27:55.6120876Z 	at io.quarkus.devui.runtime.comms.JsonRpcRouter.lambda$addSocket$1(JsonRpcRouter.java:111)
2023-07-01T02:27:55.6121485Z 	at io.vertx.core.http.impl.WebSocketImplBase$FrameAggregator.handleTextFrame(WebSocketImplBase.java:630)
2023-07-01T02:27:55.6122095Z 	at io.vertx.core.http.impl.WebSocketImplBase$FrameAggregator.handle(WebSocketImplBase.java:596)
2023-07-01T02:27:55.6122656Z 	at io.vertx.core.http.impl.WebSocketImplBase$FrameAggregator.handle(WebSocketImplBase.java:585)
2023-07-01T02:27:55.6123190Z 	at io.vertx.core.impl.ContextInternal.dispatch(ContextInternal.java:264)
2023-07-01T02:27:55.6123733Z 	at io.vertx.core.http.impl.WebSocketImplBase.receiveFrame(WebSocketImplBase.java:533)
2023-07-01T02:27:55.6124309Z 	at io.vertx.core.streams.impl.InboundBuffer.handleEvent(InboundBuffer.java:255)
2023-07-01T02:27:55.6124825Z 	at io.vertx.core.streams.impl.InboundBuffer.write(InboundBuffer.java:134)
2023-07-01T02:27:55.6125364Z 	at io.vertx.core.http.impl.WebSocketImplBase.handleFrame(WebSocketImplBase.java:475)
2023-07-01T02:27:55.6125900Z 	at io.vertx.core.impl.EventLoopContext.execute(EventLoopContext.java:76)
2023-07-01T02:27:55.6126405Z 	at io.vertx.core.impl.DuplicatedContext.execute(DuplicatedContext.java:153)
2023-07-01T02:27:55.6126972Z 	at io.vertx.core.http.impl.Http1xConnectionBase.handleWsFrame(Http1xConnectionBase.java:64)
2023-07-01T02:27:55.6127616Z 	at io.vertx.core.http.impl.Http1xServerConnection.handleOther(Http1xServerConnection.java:184)
2023-07-01T02:27:55.6128472Z 	at io.vertx.core.http.impl.Http1xServerConnection.handleMessage(Http1xServerConnection.java:169)
2023-07-01T02:27:55.6129036Z 	at io.vertx.core.net.impl.ConnectionBase.read(ConnectionBase.java:158)
2023-07-01T02:27:55.6129538Z 	at io.vertx.core.net.impl.VertxHandler.channelRead(VertxHandler.java:153)
2023-07-01T02:27:55.6130535Z 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:442)
2023-07-01T02:27:55.6131317Z 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)
2023-07-01T02:27:55.6132192Z 	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)
2023-07-01T02:27:55.6132959Z 	at io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:286)
2023-07-01T02:27:55.6133989Z 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:442)
2023-07-01T02:27:55.6134768Z 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)
2023-07-01T02:27:55.6135549Z 	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)
2023-07-01T02:27:55.6160490Z 	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:346)
2023-07-01T02:27:55.6161215Z 	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:318)
2023-07-01T02:27:55.6161932Z 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:444)
2023-07-01T02:27:55.6162829Z 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)
2023-07-01T02:27:55.6163600Z 	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)
2023-07-01T02:27:55.6164373Z 	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1410)
2023-07-01T02:27:55.6165152Z 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:440)
2023-07-01T02:27:55.6165985Z 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)
2023-07-01T02:27:55.6166777Z 	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:919)
2023-07-01T02:27:55.6167416Z 	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:166)
2023-07-01T02:27:55.6177752Z 	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:788)
2023-07-01T02:27:55.6178485Z 	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:724)
2023-07-01T02:27:55.6179186Z 	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:650)
2023-07-01T02:27:55.6179711Z 	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562)
2023-07-01T02:27:55.6180292Z 	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
2023-07-01T02:27:55.6180907Z 	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
2023-07-01T02:27:55.6181523Z 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
2023-07-01T02:27:55.6182068Z 	at java.base/java.lang.Thread.run(Thread.java:833)
2023-07-01T02:27:55.6182454Z Caused by: java.lang.reflect.InvocationTargetException
2023-07-01T02:27:55.6183018Z 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2023-07-01T02:27:55.6183695Z 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
2023-07-01T02:27:55.6184450Z 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2023-07-01T02:27:55.6185064Z 	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
2023-07-01T02:27:55.6185553Z 	at io.quarkus.devui.runtime.comms.JsonRpcRouter.lambda$invoke$0(JsonRpcRouter.java:98)
2023-07-01T02:27:55.6186212Z 	at io.smallrye.mutiny.unchecked.UncheckedSupplier.lambda$toSupplier$0(UncheckedSupplier.java:41)
2023-07-01T02:27:55.6186687Z 	... 48 more
2023-07-01T02:27:55.6187437Z Caused by: java.lang.StringIndexOutOfBoundsException: begin 0, end -1, length 19
2023-07-01T02:27:55.6188171Z 	at java.base/java.lang.String.checkBoundsBeginEnd(String.java:4602)
2023-07-01T02:27:55.6188649Z 	at java.base/java.lang.String.substring(String.java:2705)
2023-07-01T02:27:55.6189159Z 	at io.quarkus.arc.runtime.devui.ArcJsonRPCService.timeString(ArcJsonRPCService.java:103)
2023-07-01T02:27:55.6189957Z 	at io.quarkus.arc.runtime.devui.ArcJsonRPCService.toInvocationInfo(ArcJsonRPCService.java:92)
2023-07-01T02:27:55.6190762Z 	at io.quarkus.arc.runtime.devui.ArcJsonRPCService.toInvocationInfos(ArcJsonRPCService.java:84)
2023-07-01T02:27:55.6191519Z 	at io.quarkus.arc.runtime.devui.ArcJsonRPCService.getLastInvocations(ArcJsonRPCService.java:67)
2023-07-01T02:27:55.6192222Z 	at io.quarkus.arc.runtime.devui.ArcJsonRPCService_ClientProxy.getLastInvocations(Unknown Source)
```

--- a/extensions/arc/runtime/src/main/java/io/quarkus/arc/runtime/devui/ArcJsonRPCService.java
+++ b/extensions/arc/runtime/src/main/java/io/quarkus/arc/runtime/devui/ArcJsonRPCService.java
@@ -100,6 +100,7 @@ public class ArcJsonRPCService {
 
     private String timeString(LocalDateTime time) {
         String timestamp = time.format(DateTimeFormatter.ISO_LOCAL_DATE_TIME).replace("T", " ");
-        return timestamp.substring(0, timestamp.lastIndexOf("."));
+        int lastIndexOfDot = timestamp.lastIndexOf(".");
+        return lastIndexOfDot > 0 ? timestamp.substring(0, lastIndexOfDot) : timestamp;
     }
 }

